install_script.sh.template

```diff
@@ -770,7 +772,9 @@ if [ "$OS" == "RedHat" ]; then
 
     echo -e "  \033[33mInstalling package(s): ${packages[*]}\n\033[0m"
 
-    $sudo_cmd yum -y --disablerepo='*' --enablerepo='datadog' install $dnf_flag "${packages[@]}" || $sudo_cmd yum -y install $dnf_flag "${packages[@]}"
+    $sudo_cmd yum -y --disablerepo='*' --enablerepo='datadog' install $dnf_flag "${packages[@]}" 2>"/tmp/ddog_install_error_msg" || $sudo_cmd yum -y install $dnf_flag "${packages[@]}" 2>"/tmp/ddog_install_error_msg"
+    cat "/tmp/ddog_install_error_msg"
```

Comment:
Is displaying the entire log expected or is this a left over debug?

Reply:
I kinda wanted to use something similar to `tee` to store stderr into a file and still show it on stderr this way it doesn't remove the error from the live logs. I didn't find a great alternative since `tee` only works for stdout and wasn't sure if redirecting stderr into stdout would be a great idea. At the moment I'm simply storing stderr and printing it back to avoid any information loss. But still not great because it messes up the order in the log if there's an error. If you have any suggestion I'm open to it 🤔

Reply:
Forcing errors onto `stdout` might be counter intuitive indeed, although currently we don't display anything and the errors are displayed on `stdout`, so as far as which FD we use, I guess that's fine
My worry was mostly about adding the entire log file to the terminal, since that wasn't the case before, and it might be a bit too verbose, but I'll let you decide as long as the change is intentional :)